<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapExecution</class>
 <widget class="QWidget" name="MapExecution">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>67</red>
        <green>74</green>
        <blue>84</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>67</red>
        <green>74</green>
        <blue>84</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>67</red>
        <green>74</green>
        <blue>84</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>67</red>
        <green>74</green>
        <blue>84</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="12,2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="6,1,1,2">
       <item>
        <widget class="TableWidget" name="tableView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>290</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>290</width>
           <height>16777215</height>
          </size>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="DigitalClock" name="StatusIndicator">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>290</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>290</width>
           <height>50</height>
          </size>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DigitalClock" name="clock">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>290</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>290</width>
           <height>50</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DigitalClock" name="StatusConsole">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>290</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>290</width>
           <height>100</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="MapWidget" name="mapView" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="styleSheet">
        <string notr="true"> QPushButton { 
background-color: &quot;#CCD1D9&quot;; 
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #E6E9ED
 }</string>
       </property>
       <property name="text">
        <string>Start Mission</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="styleSheet">
        <string notr="true"> QPushButton { 
background-color: &quot;#CCD1D9&quot;; 
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #E6E9ED
 }</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="finishButton">
       <property name="styleSheet">
        <string notr="true"> QPushButton { 
background-color: &quot;#CCD1D9&quot;; 
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #E6E9ED
 }</string>
       </property>
       <property name="text">
        <string>Finish</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DigitalClock</class>
   <extends>QPlainTextEdit</extends>
   <header>digitalclock.h</header>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableView</extends>
   <header>tablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>MapWidget</class>
   <extends>QWidget</extends>
   <header>mapwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>slot1()</slot>
  <slot>finishClicked()</slot>
  <slot>returnHomeClicked()</slot>
  <slot>cancelClicked()</slot>
 </slots>
</ui>
